lib/bank_account/saving_account.rb

```diff
@@ -0,0 +1,30 @@
+module BankAccount
+  class SavingAccount < Account
+
+    def initialize(id, initial_balance)
+      # SavingAccount inherits from acount, initialize method is exactly the same
+      super
+    end
+
+    def withdraw(amount)
+
+      # savings account cannot dip below $10
+      if @balance - amount < 10
+        puts "Your balance cannot be less than $10 in your savings account."
+      else
+        super
```

Comment:
Excellent user of `super` here. You identified what the varying behavior is (minimum balance requirement) and handled the logic well. Great job.